src/deploy-cromwell-on-azure/scripts/startup.sh

```diff
@@ -117,15 +116,15 @@ kv["TesImageSha"]=$(docker inspect --format='{{range (.RepoDigests)}}{{.}}{{end}
 kv["TriggerServiceImageSha"]=$(docker inspect --format='{{range (.RepoDigests)}}{{.}}{{end}}' ${kv["TriggerServiceImageName"]})
 rm -f .env && for key in "${!kv[@]}"; do echo "$key=${kv[$key]}" >> .env; done
 
-if [ -n "$mysql_server_name" ]; then
-    fully_qualified_server_name="$mysql_server_name.mysql.database.azure.com"
-    mysql_db_name=${kv["MySqlDatabaseName"]}
-    mysql_user_login=${kv["MySqlUserLogin"]}
-    mysql_user_password=${kv["MySqlUserPassword"]}
-    write_log "Checking if database $mysql_db_name is locked"
-    lockTableExists=$(mysql -u "$mysql_user_login" -p"$mysql_user_password" -h "$fully_qualified_server_name" -P 3306 -e "SELECT EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE table_schema = '$mysql_db_name' AND table_name = 'DATABASECHANGELOGLOCK') AS '' ")
-    [[ "$lockTableExists" -eq "1" ]] && isLocked=$(mysql -u "$mysql_user_login" -p"$mysql_user_password" -h "$fully_qualified_server_name" -P 3306 -e "SELECT CONV(LOCKED, 2, 2) AS '' FROM $mysql_db_name.DATABASECHANGELOGLOCK WHERE ID = 1") || isLocked=0
-    [[ "$isLocked" -eq "1" ]] && write_log "Removing lock from $mysql_db_name" && mysql -u "$mysql_user_login" -p"$mysql_user_password" -h "$fully_qualified_server_name" -P 3306 -e "UPDATE $mysql_db_name.DATABASECHANGELOGLOCK SET LOCKED = 0, LOCKGRANTED = null, LOCKEDBY = null WHERE ID = 1" || write_log "$mysql_db_name was not locked"
+if [ -n "$db_server_name" ]; then
+    fully_qualified_server_name="$db_server_name.postgres.database.azure.com"
+    db_name=${kv["PostgreSqlDatabaseName"]}
+    db_user=${kv["PostgreSqlUserLogin"]}
+    db_user_password=${kv["PostgreSqlUserPassword"]}
+    write_log "Checking if database $db_name is locked"
```

Comment:
This works
psql -t -d "host="$fully_qualified_server_name" dbname=$db_name user=$db_user password=$db_user_password"
but I'd still adjust the quotes for the host:
psql -t -d "host='$fully_qualified_server_name' dbname=$db_name user=$db_user password=$db_user_password"

Reply:
thank you- switched to using single quotes